src/nxdoc/web-ui/web-ui-release-notes/web-ui-release-notes-3-1-2.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Version 3.1.2
+description: Discover what's new in Web UI 3.1.2.
+review:
+    comment: ''
+    date: '2023-07-07'
+    status: ok
+toc: true
+labels:
+tree_item_index: 999
+hidden: true
+---
+
+{{{multiexcerpt 'matching-notes' page='web-ui-release-notes'}}}
+
+{{! multiexcerpt name='web-ui-updates'}}
+## What’s New in Web UI for LTS 2023 (Version 3.1.2)
+
+This release is based upon [Web UI 3.0.24 for LTS 2021]({{page page='web-ui-release-notes-3-0-24' version='2021'}}).
```

Comment:
@poonamyadav252 I'm not sure if this part is accurate or not, if you can help me with this it would be very appreciated!

Reply:
@bchauvin Yes, It is the correct version.